crates/bevy_render/src/renderer/mod.rs

```diff
@@ -135,8 +135,7 @@ pub async fn initialize_renderer(
 
     // Maybe get features and limits based on what is supported by the adapter/backend
     let mut features = wgpu::Features::empty();
-    let mut limits = options.limits.clone();
-    if matches!(options.priority, WgpuSettingsPriority::Functionality) {
+    let mut limits = if matches!(options.priority, WgpuSettingsPriority::Functionality) {
```

Comment:
I think this would be clearer as a match statement.

Reply:
I don't think I'm too happy with either version, but you may want to look at this diff in context.  Easier/safer just not to make any change to this code.

Reply:
Yeah, I think we don't touch this code in this PR.

Reply:
I can't seem to find the button that reverts this particular change.  Or do I do it on the commit for this file?

Reply:
Yeah, you'll want to revert the commit for this file :)

Reply:
Reverted.